src/libraries/Common/src/Interop/Windows/SspiCli/SSPIWrapper.cs

```diff
@@ -141,6 +141,31 @@ public static SafeFreeCredentials AcquireCredentialsHandle(ISSPIInterface secMod
             return outCredential;
         }
 
+        public static unsafe SafeFreeCredentials AcquireCredentialsHandle(ISSPIInterface secModule, string package, Interop.SspiCli.CredentialUse intent, Interop.SspiCli.SCH_CREDENTIALS* scc)
+        {
+            if (NetEventSource.IsEnabled)
+            {
+                NetEventSource.Enter(null, package);
```

Comment:
Have you found such Enter/Exit tracing to be valuable?  I'm hoping we might actually rip them all out.  I realize you're just following the same pattern that's used elsewhere, so it's fine, but... yeah.

Reply:
I did no not - so far. But I'm not the PerfView master. I added them for consistency. 
BTW you can filter them out easily with pipeline keywords. As far as I know, with the right filter, the entry/exit would not be edited - but it would still stay in source code and we would need to maintain it.

Reply:
> but it would still stay in source code and we would need to maintain it.

Yes.  It's extra maintenance (and we're already wildly inconsistent on it) and extra overhead.

Reply:
since the wrapper really does not do anything I took it out and did the same to legacy function.